simapp/app_v2.go

```diff
@@ -3,7 +3,6 @@
 package simapp
 
 import (
-	_ "embed"
```

Comment:
is this required?

Reply:
Nope, we import it in `client/docs` already, so it can be removed.